src/mbedtls/Makefile.in

```diff
@@ -0,0 +1,33 @@
+@SET_MAKE@
+CC = @CC@
+AS = @CC@
+LD = @CC@
+CPP = @CC@
+CFLAGS = @CFLAGS@ @CFLAGS_EXTRA@ @JOHN_NO_SIMD@ -std=c99
```

Comment:
I'm not happy about explicit `-std=c99` that is not part of `CFLAGS`, but I see we already have it in a number of places, so this is something we'll want to move to global scope separately (perhaps make it part of `CFLAGS` where supported).

Reply:
It's probably about time we do that globally. OpenCL lures me into c99 style :)